llvm/include/llvm/AsmParser/LLToken.h

```diff
@@ -482,6 +483,8 @@ enum Kind {
   // Type valued tokens (TyVal).
   Type,
 
+  DbgRecordType,
```

Comment:
I think actually these need to be moved to the "String valued tokens" above, since they follow the same model (a `DbgRecordType` token is just the text "dbg_foo").